inc/config.php

```diff
@@ -156,18 +175,26 @@
             exit();
         }
     }
-   // Check version
-    if (
-        (!isset($CFG_GLPI['dbversion']) || (trim($CFG_GLPI["dbversion"]) != GLPI_SCHEMA_VERSION))
-        && !isset($_GET["donotcheckversion"])
-    ) {
+    // Check version
+    $skip_version_checks = isset($_GET["donotcheckversion"]);
+    $has_known_version   = isset($CFG_GLPI['dbversion']);
+    $can_check_version   = is_readable(GLPI_ROOT . '/install/mysql/glpi-empty.sql');
+    if (!$skip_version_checks && $has_known_version && !$can_check_version) {
+        trigger_error(
```

Comment:
As discussed with @trasher, maybe we can remove this warning. If the `/install/mysql/glpi-empty.sql` is missing, we could just ignore the DB update check.